package.json

```diff
@@ -23,14 +23,18 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "bignumber.js": "^9.0.1",
     "bn.js": "^5.2.0",
     "buffer": "^6.0.3",
+    "cbor": "^7.0.5",
```

Comment:
I'm sorry I'm not sure about this one. It seems it's dependent on Node.js `stream` I tried to install `cbor` in RN app but getting the following error:
![image](https://user-images.githubusercontent.com/676169/119789397-4fbb8b80-bed3-11eb-8030-c9543dbc4c83.png)

Reply:
The library is called node-cbor so good chance it won't work in RN

Reply:
Ah sorry, forgot about cbor. I will find an alternative.

Reply:
Changed the dependency to `borc`. Tested it in React Native and it works for me.

Reply:
Great 😄